### ISSUE TITLE
[BUG] Image's alt text should be announced by screen readers

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5439]
**Describe the bug**
When a TalkBack user is navigating a Group or event with an image, they can double click to zoom on image. The image's alt text from the new alt text field in #4836 should be announced by Talkback. 

**To Reproduce**
1. Go to "Groups" or "Events"
2. Select a photo

**Expected behavior**
The image's alt text from the new alt text field in #4836 should be announced by Talkback. 

**Actual behavior*
The image is selected and the TalkBack user hears a bmp sound. (7.3.8 version of apk)

**Screenshots**

https://github.com/user-attachments/assets/9c5021bf-e251-491f-a4f2-6ebcc51c3f82

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**
• OS: Android Version 15
• Device: Google Pixel 8a
APK Version 7.3.8

**Additional context**

**Comment by acovher2** [comment]
Retested and passed in Android Pixel 8a 7.3.21
